src/config/_inputs.scss

```diff
@@ -0,0 +1,25 @@
+
+// Dependencies
+@import 'config/variables';
+@import 'config/colors';
+@import 'config/fonts';
+
+$padding: map-get($variables, 'padding');
+
+// Declarations
+@mixin inputs() {
+  color: $color-base-black;
+  padding: .75 * map-get($padding, 2);
+  font-size: 18px;
```

Comment:
font size should be `inherit`

Reply:
Also, if we were to use `px` values in the styles for font sizes, this would break the ability to use a custom text resizer to resize them... although... I don't know why we would do that here (like ACCESS) but it is a useful pattern for Accessibility. Use font size variables in the config (which are `rem` units).